### PR TITLE
Add code samples for PHP

### PR DESCRIPTION
**Pull request by withinboredom**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Adds PHP code samples where appropriate.

## Issue reference

#1196

### CODE REVIEW

**Comment by orizohar** [comment]
@withinboredom - can you please replace this PR with one that goes into v1.0 branch? (rc3 branch will be removed from the website very soon...)

**Comment by withinboredom** [comment]
rebasing

**Comment by withinboredom** [comment]
@orizohar rebased and retargeted the 1.0 branch.

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state.md [hunk 1]
```diff
@@ -400,6 +531,57 @@ You're up and running! Both Dapr and your app logs will appear here.
 
 {{% /codetab %}}
 
+{{% codetab %}}
+
+Transactional state is supported by extending `TransactionalState` base object which hooks into your
+object via setters and getters to provide a transaction. Before we created our own transactional object,
```
Comment (orizohar) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state.md]: Please avoid 1st person "we", "our" as described [here](https://docs.dapr.io/contributing/contributing-docs/#style-and-tone)

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state.md [hunk 2]
```diff
@@ -400,6 +531,57 @@ You're up and running! Both Dapr and your app logs will appear here.
...
+but now we'll ask the Dependency Injection framework to build one for us.
```
Comment (orizohar) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-get-save-state.md]: [Same guidance](https://docs.dapr.io/contributing/contributing-docs/#style-and-tone) goes for "we'll", "us"

**Review by orizohar — CHANGES_REQUESTED** [review]
Thanks for this! Just a minor comment on usage of 1st person in the docs

**Review by orizohar — APPROVED** [review]

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**